c10/util/complex_math.h

```diff
@@ -9,7 +9,7 @@ namespace std {
 #endif
 
 template<typename T>
-C10_HOST_DEVICE c10::complex<T> exp(c10::complex<T> x) {
+C10_HOST_DEVICE inline c10::complex<T> exp(const c10::complex<T> &x) {
```

Comment:
why do all these functions have to be made inline?

Reply:
It doesn't have to be inline, I just feel inline is better since these functions are small.

Reply:
I think it makes sense to inline here because these functions would be heavily used (in loops in kernels)